CONTRIBUTOR.md

```diff
@@ -0,0 +1,97 @@
+# Contributing to Layer0
+
+This page contains information about reporting issues as well as some tips and
+guidelines for potential open source contributors. Make sure you read our [community guidelines](#layer0-community-guidelines) before participating.
+
+If you're interested in hacking on Layer0, please also see our [README.md](README.md),
+which details the various components of Layer0.
+
+Our contributor documentation is based largely on the good work at
+[Docker](https://github.com/docker/docker). Thanks to the Docker team for helping
+to establish a solid OSS community!
+
+## Topics
+
+* [Reporting Security Issues](#reporting-security-issues)
+* [Reporting Issues](#reporting-other-issues)
+* [Quick Contribution Tips and Guidelines](#quick-contribution-tips-and-guidelines)
+* [Community Guidelines](#layer0-community-guidelines)
+
+## Reporting security issues
+
+Please **DO NOT** file a public issue, instead send your report(s) privately to
+[carbon@us.imshealth.com](mailto:carbon@us.imshealth.com).
+
+## Reporting other issues
+
+The easiest way to contribute to Layer0 is to send us a detailed report when you
+encounter an issue.
+
+Check that [our issue database](https://github.com/quintilesims/layer0/issues)
+doesn't already include your problem or suggestion before submitting an issue.
+If you find a match, you can use the "subscribe" button to get notified on
+updates.
+
+When reporting issues, include (where applicable):
+
+* The output of `l0 --version`.
+* The output of `l0 admin debug`.
+* The output of `l0-setup --version`
+
+Also include the steps required to reproduce the problem if possible.
```

Comment:
Be specific in the problem(s) you are encountering.  It could be helpful to also include what you're trying to do, so we can have a better grasp of context when prioritizing our work.